### DISCUSSION TITLE
Call to undefined function Livewire\tmpfile()

### DISCUSSION THREAD

**Discussion by ejulfaey** [livewire/livewire#4495] [1. Help]
Hi, I have this kind of error only in production server while my local is working perfectly. Thanks.

<img width="1261" alt="Screenshot 2022-01-09 at 16 10 00" src="https://user-images.githubusercontent.com/15130147/148674503-211b6d59-af98-4e88-b577-9f79fadf8efa.png">

**Comment by joshhanley** [comment]
This issue has been converted to a discussion as we are now using discussions to track bugs and ideas.
See this announcement for more details https://github.com/livewire/livewire/discussions/3147

**Comment by joaopalopes24** [comment]
Did you find the problem?

**Comment by Ricardo-Lizarazo** [comment]
Modifica el archivo TemporaryUploadedFile de Livewire

Cambia de esto:
$tmpFile = tmpfile();

A esto:
$tmpfname = tempnam(sys_get_temp_dir(), '');
$tmpFile = fopen($tmpfname, "w");

**Reply by AleisterFox** [reply]
Me salvaste
